src/index.ts

```diff
@@ -7,8 +7,12 @@ import fs from 'fs'
 async function main(): Promise<void> {
   const { token, title, path } = getInputs()
 
-  const github = new Github(token)
+  if (!fs.existsSync(path)) {
+    core.warning(`Input file ${path} does not exist`)
+    return
```

Comment:
`process.exit(1)`?

Reply:
If it was a node thing yup; but as it's an action we don't want to exit the process.